[logs] --search does not support spaces

It looks like passing a phrase to `--search` (with spaces) causes 401.

```bash
cc-api on  ensureProvisioningOnce [$?] via ☕ v17.0.8 
➜ clever logs -a prod --search "Thread"   
Waiting for application logs…
2024-02-12T14:06:43.866Z: [2024-02-12T14:06:43.496+0000] [blabla]
^C%                                                                                                                                                                                                                                                                                                                                                         

cc-api on  ensureProvisioningOnce [$?] via ☕ v17.0.8 took 2s 
➜ clever logs -a prod --search "Thread o" 
Waiting for application logs…
[ERROR] HTTP error 401: Unauthorized to access this resource.
```

Using `clever curl`

I have no issue when I'm URL encoding the query string value:

```bash
clever  curl "https://api.clever-cloud.com/v4/logs/organisations/orga_8b852f0a-1135-4b4f-8eee-a538f538b640/applications/app_6ff249b9-3c69-431c-b356-f4c327c03d40/logs?filter=EventsListener%20-%20Thread"
IS OK
```

--- a/package.json
+++ b/package.json
@@ -25,7 +25,7 @@
     "scripts/*.sh"
   ],
   "dependencies": {
-    "@clevercloud/client": "^8.1.0",
+    "@clevercloud/client": "^8.1.2",
     "clf-date": "^0.2.0",
     "cliparse": "^0.3.3",
     "colors": "1.4.0",

